pkg/monitor/types.go

```diff
@@ -31,3 +40,73 @@ const (
 	// MessageTypeAgent is an agent notification carrying a AgentNotify
 	MessageTypeAgent = 130
 )
+
+var (
+	names = map[string]int{
+		"drop":    MessageTypeDrop,
+		"debug":   MessageTypeDebug,
+		"capture": MessageTypeCapture,
+		"trace":   MessageTypeTrace,
+		"l7":      MessageTypeAccessLog,
+		"agent":   MessageTypeAgent,
+	}
+)
+
+func type2name(typ int) string {
+	for name, value := range names {
+		if value == typ {
+			return name
+		}
+	}
+
+	return strconv.Itoa(typ)
+}
+
+type MessageTypeFilter []int
+
+var _ pflag.Value = &MessageTypeFilter{}
+
+func (m *MessageTypeFilter) String() string {
+	pieces := make([]string, 0, len(*m))
+	for _, typ := range *m {
+		pieces = append(pieces, type2name(typ))
+	}
+
+	return strings.Join(pieces, ", ")
```

Comment:
I would remove the space, to make it easier to use. Like K8s label selector strings.

Reply:
fixed